### ISSUE TITLE
Add Ente icon

### ISSUE THREAD

**Issue by skariko** [jamie-mh/AuthenticatorPro#519]
Hello,
Ente is a privacy cloud-storage alternative to Google Photos.

The website sit [ente.io](https://ente.io/) and I have created the icon for Authenticator Pro based on their Twitter account: https://twitter.com/enteio

This is the icon:
![ente](https://user-images.githubusercontent.com/25532821/189671206-601e8f70-6cc7-4c47-b5b7-a39df3457e68.png)

Thanks.

**Comment by skariko** [comment]
Sorry I only now read the FAQ regarding icons.

Unfortunately Ente is not among the top 200 thousand sites and not have 100k+ download on Play Store, but I think it is a fairly well-known privacy alternative among users who also use Authenticator Pro.

If you think an exception can be made you can use the icon I created otherwise go ahead and close this issue.

Thank you!

**Comment by RobertHarnett** [comment]
I'll more then happily add this, there is very much a overlap with our users, and it appears on [privacytools.io](https://www.privacytools.io/) so it's got the green light from me. I mainly created the requirements because I noticed that the icons in the app were slowly getting bloated with dozens of niche and obscure crypto sites. This is the kind of service I will more then happily make an exception for. This can be closed as I have added it in #521.
